beets/library.py

```diff
@@ -747,6 +747,20 @@ def move_file(self, dest, operation=MoveOperation.MOVE):
             util.hardlink(self.path, dest)
             plugins.send("item_hardlinked", item=self, source=self.path,
                          destination=dest)
+        elif operation == MoveOperation.REFLINK:
+            util.reflink(self.path, dest, fallback=False)
+            plugins.send("item_reflinked", item=self, source=self.path,
+                         destination=dest)
+        elif operation == MoveOperation.REFLINK_AUTO:
+            util.reflink(self.path, dest, fallback=True)
+            plugins.send("item_reflinked", item=self, source=self.path,
```

Comment:
Maybe only send if the reflink was successful? I am thinking of the case where it fails, and a copy is used instead. This signal would seem to indicate a reflink was created when it actually was not.

Reply:
Hmm, I see why, but that will complicate things even more, I think. I notice most of the other operations have a `before_` signal too, and maybe it might mean `before_item_reflinked` would go before a `item_copied`.

Not sure how to proceed here.  `util.reflink` does not return whether it had to fall-back or not either, but that's an easy addition.

Reply:
Yeah, I think this is OK as is for now.

Reply:
yeah, my initial thought was something like:

```
        elif operation == MoveOperation.REFLINK_AUTO:
            try:
                util.reflink(self.path, dest)
            except (NotImplementedError, pyreflink.ReflinkImpossibleError):
                self.move_file(self, dest, MoveOperation.COPY)
            else:
                plugins.send(
                    "item_reflinked", item=self, source=self.path, destination=dest
                )
```
removing the need for the fallback parameter